### PR TITLE
First pass files api for docker

### PR DESCRIPTION
**Pull request by hardillb**
## Description

Adds the driver interface to https://github.com/FlowFuse/nr-launcher/issues/273

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by hardillb** [comment]
@Steve-Mcl Like k8s, please review but not merge

**Comment by hardillb** [comment]
Localfs for comparison https://github.com/FlowFuse/driver-localfs/pull/144

File: docker.js [hunk 1]
```diff
@@ -170,6 +171,10 @@ const createContainer = async (project, domain) => {
         })
 }
 
+const getStaticFileUrl = async (project, filePath) => {
+    return `http://${project.id}:2880/flowforge/files_/${filePath}`
```
Comment (Steve-Mcl) [inline docker.js:175]: should that be `/_/`?

Comment (Steve-Mcl) [inline docker.js:175]:
```suggestion
const getStaticFileUrl = async (instance, filePath) => {
    return `http://${instance.id}:2880/flowforge/files_/${filePath}`
```

keep with instance naming as in other new functions.

**Review by Steve-Mcl — CHANGES_REQUESTED** [review]

File: docker.js (hunk 1 above)
Comment (hardillb) [reply docker.js]: I had fixed this but not pushed it

**Review by Steve-Mcl — APPROVED** [review]

**Pull request merged**